keras/src/ops/core.py

```diff
@@ -802,6 +802,67 @@ def cast(x, dtype):
     return backend.core.cast(x, dtype)
 
 
+class SaturateCast(Operation):
+    def __init__(self, dtype):
+        super().__init__()
+        self.dtype = backend.standardize_dtype(dtype)
+
+    def call(self, x):
+        return _saturate_cast(x, self.dtype)
+
+    def compute_output_spec(self, x):
+        return backend.KerasTensor(shape=x.shape, dtype=self.dtype)
+
+
+@keras_export("keras.ops.saturate_cast")
+def saturate_cast(x, dtype):
+    """Performs a safe saturating cast to the desired dtype.
```

Comment:
Please add information about what a saturate cast is and when users should want to do it.